plugins/modules/dcnm_interface.py

```diff
@@ -4567,6 +4619,16 @@ def dcnm_intf_send_message_to_dcnm(self):
         path = self.paths["INTERFACE"]
         for payload in self.diff_replace:
 
+            if payload.get('policy') == 'int_mpls_loopback':
```

Comment:
@mmudigon if there is a better place or way to add these properties to the payload let me know what you think.  This seems fairly simple/safe but there could be a better option.